### ISSUE TITLE
Phishlets found now ?

### ISSUE THREAD

**Issue by Dazmed707** [klezVirus/evilginx-collection#1]
U tested this phishlets in evilginx 2.4.3 ?
Found Google phishlet?

**Comment by klezVirus** [comment]
Nope, I just collected stuff that I wanted to test so that I can automate the cloning process. TLDR; the phishlets here are the ones I'm testing now. I will remove / add stuff that I find convenient on the go.
RE: Google - As long as I know Phishlets do not work well against captcha v3, but I've yet to do some tests against it.

**Comment by Dazmed707** [comment]
U can update Google phishlet this week ?

**Comment by xday3** [comment]
Do you have any facebook phishlet that is working?

**Comment by evilginx3** [comment]
I have updated my repo with google phishlet, check it out
